virtcontainers/sandbox_test.go

```diff
@@ -1748,3 +1749,82 @@ func TestStartNetworkMonitor(t *testing.T) {
 	err = s.startNetworkMonitor()
 	assert.Nil(t, err)
 }
+
+func checkDirNotExist(path string) error {
+	if _, err := os.Stat(path); os.IsExist(err) {
+		return fmt.Errorf("%s is still exists", path)
+	}
+	return nil
+}
+
+func TestSandboxCreationRollback(t *testing.T) {
+	defer cleanUp()
```

Comment:
Do not defer this `cleanUp()`, just call before everything else in this function.

Reply:
updated~